src/detect_breakends/junction_detection.cpp

```diff
@@ -365,6 +369,20 @@ void detect_junctions_in_alignment_file(const std::filesystem::path & alignment_
             break;
     }
 
+    switch (refinement_method)
+    {
+        case 0: // no refinement
+            seqan3::debug_stream << "No refinement was selected.\n";
+            break;
+        case 1: // sViper
+            seqan3::debug_stream << "The sViper refinement method is not jet implemented\n";
```

Comment:
```suggestion
            seqan3::debug_stream << "The sViper refinement method is not yet implemented\n";
```

Reply:
and I did it again ^^ thanks for the hint 😅